manager/integration/tests/test_ha.py

```diff
@@ -1242,6 +1243,8 @@ def test_single_replica_restore_failure(client, core_api, volume_name, pod_make)
     res_volume = client.by_id_volume(res_name)
     assert res_volume.ready is False
 
+    res_volume = wait_for_volume_healthy_no_frontend(client, res_name)
+
     crash_replica_processes(client, core_api, res_name,
```

Comment:
We better add an `assert` in `crash_replica_processes` and similar functions that the instance manager must exist before continue. It can be a separate commit.

Reply:
fixed in a separate commit.

Reply:
hmm? which one?